corehq/apps/reports/generic.py

```diff
@@ -943,10 +943,11 @@ def report_context(self):
             context.update(provider_function(self))
         return context
 
-    def table_cell(self, value, html=None):
+    def table_cell(self, value, html=None, zerostyle=False):
+        styled_value = '<span class="gray-zero">0</span>' if zerostyle and value == 0 else value
```

Comment:
I never figured out why I didn't see the changes locally, but happy to try as soon as you've updated it.